workers/unity/Packages/com.improbable.gdk.transformsynchronization/Systems/TakeRawTransformUpdateSystem.cs

```diff
@@ -24,7 +24,7 @@ private struct Data
 
         protected override void OnUpdate()
         {
-            for (int i = 0; i < data.Length; ++i)
+            for (var i = 0; i < data.Length; ++i)
```

Comment:
I don't actually like var in the transform stuff as a lot of these are over uints rather than int. Although I don't care that much.

Reply:
I don't really care much either, though our current code settings do prefer it